pkg/controllers/secretstore/common.go

```diff
@@ -67,6 +67,15 @@ func reconcile(ctx context.Context, req ctrl.Request, ss esapi.GenericStore, cl
 		log.Error(err, "unable to validate store")
 		return ctrl.Result{}, err
 	}
+	storeProvider, err := esapi.GetProvider(ss)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	capStatus := esapi.SecretStoreStatus{
+		Capabilities: storeProvider.Capabilities(),
```

Comment:
So, this was something that we discussed in the community meeting. I am still not sure about this Capabilities column as we are using it right now. Right now it is just a "info" column, and I think the intention is for it to be a column informing the user if this Write is implemented for a provider. I think it would be super interesting to have a way to control if a SecretStore is Read or Write or ReadWrite with probably a new field in the SecretStore and then just report it here. Then we would have provider specific logic to implement this (probably for a folowup PR, but worth discussing what we should do with the column in the mean time).

Reply:
I agree with you, this makes a lot of sense, specially when it comes to permission boundaries. Another PR for it would be better indeed.

IMO this column is still valid as is, even if we provide the feature to configure if a SecretStore can be used as a Write one.